1

```diff
@@ -0,0 +1,16 @@
+docs(icons): add icons documentation
```

Comment:
I think that you should rename this file?

Reply:
Most def remove it lol